monkestation/code/modules/art_sci_overrides/asteroids/asteroid_magnet.dm

```diff
@@ -0,0 +1,216 @@
+		<legend class='computerLegend'>
+			<b>Magnet Controls</b>
+		</legend>
+	"}
+
+	// X selector
+	content += {"
+		<fieldset class='computerPaneNested'>
+			<legend class='computerLegend' style='margin: auto'>
+				<b>X-Axis</b>
+			</legend>
+			<div style='display: flex; justify-content: center; gap: 5px'>
+				<div style='display: inline-block'>[button_element(src, "-100", "x=-100")]</div>
+				<div style='display: inline-block'>[button_element(src, "-10", "x=-10")]</div>
+				<div style='display: inline-block'>[button_element(src, "-1", "x=-1")]</div>
+				<div style='display: inline-block; padding: 0.5em'>
+					<span class='computerLegend'>[coords_x]</span>
+				</div>
+				<div style='display: inline-block'>[button_element(src, "1", "x=1")]</div>
+				<div style='display: inline-block'>[button_element(src, "10", "x=10")]</div>
+				<div style='display: inline-block'>[button_element(src, "100", "x=100")]</div>
+				<span style='visibility: hidden'>---</span>
+			</div>
+		</fieldset>
+	"}
+
+	// Y selector
+	content += {"
+		<fieldset class='computerPaneNested'>
+			<legend class='computerLegend' style='margin: auto'>
+				<b>Y-Axis</b>
+			</legend>
+			<div style='display: flex; justify-content: center'>
+				<div style='display: inline-block'>[button_element(src, "-100", "y=-100")]</div>
+				<div style='display: inline-block'>[button_element(src, "-10", "y=-10")]</div>
+				<div style='display: inline-block'>[button_element(src, "-1", "y=-1")]</div>
+				<div style='display: inline-block; padding: 0.5em'>
+					<span class='computerLegend'>[coords_y]</span>
+				</div>
+				<div style='display: inline-block'>[button_element(src, "1", "y=1")]</div>
+				<div style='display: inline-block'>[button_element(src, "10", "y=10")]</div>
+				<div style='display: inline-block'>[button_element(src, "100", "y=100")]</div>
+				<span style='visibility: hidden'>---</span>
+			</div>
+		</fieldset>
+	"}
+
+	// Ping button
+	content += {"
+		<fieldset class='computerPaneNested'>
+			<legend class='computerLegend' style='margin: auto;'>
+				<b>Ping</b>
+			</legend>
+			<div class='computerLegend' style='margin: auto; width:25%'>
+				[ping_result]
+			</div>
+			<div style='margin: auto; width: 10%'>
+				[button_element(src, "PING", "ping=1")]
+			</div>
+		</fieldset>
+	"}
+
+	// Close coordinates fieldset
+	content += "</fieldset>"
+
+	// Asteroids list fieldset
+	content += {"
+	<fieldset class='computerPane'>
+		<legend class='computerLegend'>
+			<b>Available Asteroids</b>
+		</legend>
+	"}
+
+	content += {"
+		<table class='zebraTable' style='min-width:100%;height: 560px;overflow-y: auto'>
+	"}
+
+	for(var/datum/mining_template/template as anything in available_templates)
+		content += {"
+					<tr class='highlighter' style='display: block;min-width: 100%' onclick='byondCall([ref(template)])'>
+						<td>
+						<span class='computerText' style='padding-left: 10px'>[template.name] ([template.x],[template.y])</span>
```

Comment:
Please use span procs instead.

Reply:
not for raw html stuff

Reply:
the procs are micro defines, they are just referred to as procs for some reason.